webpack.mix.js

```diff
@@ -13,3 +13,8 @@ const { mix } = require('laravel-mix');
 
 mix.js('resources/assets/js/app.js', 'public/js')
    .sass('resources/assets/sass/app.scss', 'public/css');
+
+
+mix.copy('vendor/swagger-api/swagger-ui/dist/swagger-ui.css', 'public/swagger/asset/swagger-ui.css');
```

Comment:
It shouldn't be needed to copy these files over. They are automatically handled by l5-swagger

Reply:
It seems like the routes are registered. But when I remove the assets I get 404's on the requests that try to fetch the assets. I'll try debugging more and see if I can find the root cause

Reply:
https://github.com/DarkaOnLine/L5-Swagger/issues/75 references the issue I had and the solution has also been posted. I had to remove the lines from my nginx config. I'm going to remove the webpack code that I wrote.